test/test_helper.rb

```diff
@@ -36,6 +36,11 @@
     upstream: "#{SemianConfig["mysql_host"]}:#{SemianConfig["mysql_port"]}",
     listen: "#{SemianConfig["toxiproxy_upstream_host"]}:#{SemianConfig["mysql_toxiproxy_port"]}",
   },
+  {
+    name: "semian_test_postgres",
```

Comment:
This needs to be `semian_test_postgresql` because the code to bring back semian on toxiproxy down matches on the `ActiveRecordConnectionAdapter.db_client` which will return `postgresql` and then this won't match (ie `"semian_test_postgres".match?("postgresql")` will be false).